pxtblocks/blocklyloader.ts

```diff
@@ -2635,6 +2636,29 @@ namespace pxt.blocks {
                 setOutputCheck(this, typeName, cachedBlockInfo);
             };
         }
+
+        /**
```

Comment:
Is this chunk generated? This seems like kind of a weird place to preserve the jsdoc comment.

Reply:
No, we just usually preserve those in this file